docs/src/man/getting_started.md

```diff
@@ -634,7 +634,7 @@ julia> df
 ```
 
 `transform` and `transform!` functions work identically to `select` and `select!` with the only difference that
-they retain all columns that are present in the source data frame, for example:
+they retain all columns that are present in the source data frame. Here are some more advanced examples:
```

Comment:
Can we add a `rowmean`-esque example in these docs?

Reply:
added - it is showing the ability to infer collection type even if all elements are `missing` as you wanted.

Reply:
However, this functionality is not for Julia 1.0 users :(.

Reply:
added